### PR TITLE
Jenkinsfile: Make showing the stack trace optional

### PR DESCRIPTION
**Pull request by sschuberth**
As it might be distracting to see for exceptions that are handled
internally.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -96,6 +96,12 @@ pipeline {
             choices: ['--info', '--debug', '']
         )
 
+        booleanParam(
```
Comment (mnonnenmacher) [inline Jenkinsfile:99]: Commit message: "optional", "to see exceptions"

File: Jenkinsfile [hunk 2]
```diff
@@ -96,6 +96,12 @@ pipeline {
...
+            name: 'STACKTRACE',
+            defaultValue: false,
+            description: 'Show stacktrace on all exceptions'
```
Comment (mnonnenmacher) [inline Jenkinsfile:102]: Maybe "of"?

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (sschuberth) [reply Jenkinsfile:99]: Done.

File: Jenkinsfile (hunk 2 above)
Comment (sschuberth) [reply Jenkinsfile]: Done.

**Review by mnonnenmacher — APPROVED** [review]

**Pull request merged**